Equilobe.DailyReport.BL/Jira/JiraApiUrls.cs

```diff
@@ -78,6 +78,16 @@ public static string WorkLogs(string projectKey, string author, string fromDate,
             return string.Format("project = {0} AND worklogAuthor = '{1}' AND worklogDate >= '{2}' AND worklogDate <= '{3}'", projectKey, author, fromDate, endDate);
         }
 
+        public static string AllWorklogs(string authors, string startDate, string endDate)
+        {
```

Comment:
What is the difference between this and the other method that retrieves the worklogs?

Reply:
I have multiple users. As you can see, it's not worklogAuthor = {1}, but worklogAuthor in {1}, and the latter is an array of usernames.

Reply:
Ok. In that case, AllWorklogs should better be renamed to WorklogsForUserList or something like that. And eventually let's name the old method WorklogsForUser.